### DISCUSSION TITLE
Login screen broken

### DISCUSSION THREAD

**Discussion by noahfraiture** [prasanthrangan/hyprdots#987] [Q&A]
Hello I have a pretty dumb situation. Yesterday I updated my computer and my login screen came back to normal. I search to fix but I don't even find what is its name. Swaylock seems to only be for the lock wheel and wlogout seems to be the menu to shutdown or suspend. 
In what direction should I look ?

**Comment by kRHYME7** [comment]
sddm. This is already reported multiple times.  Check the open issues and Discussions. You can try to downgrade your sddm if the default log in theme looks bothering 

``` sudo downgrade sddm ```


Here's the discussion https://github.com/prasanthrangan/hyprdots/discussions/971

**Comment by noahfraiture** [comment]
Thank you !
